front/src/components/LoginPanel/comp/RegisterForm.js

```diff
@@ -33,10 +37,21 @@ const RegisterForm = (props) => {
         setPassword2(event.target.value);
     };
 
+    const handleEmailChange = (event) => {
+        setEmail(event.target.value);
+    };
+
+    const handlereasonForCreatingAccount = (event) => {
```

Comment:
Fix typo in function's name
```suggestion
    const handleReasonForCreatingAccount = (event) => {
```